### PR TITLE
Update H610 Pro configuration

### PR DESCRIPTION
**Pull request by Mrcubix**
# Verification

[Discord](https://discord.com/channels/615607687467761684/615611007951306863/776255435647746048)

Tested and working in VC after slapping windows with replug

### CODE REVIEW

File: OpenTabletDriver/Configurations/Huion/H610 Pro.json [hunk 1]
```diff
@@ -15,20 +15,47 @@
...
       "OutputInitReport": null,
       "DeviceStrings": {
-        "200": "HVAN",
-        "121": "HA60"
+        "201": "HUION_T175_\\d{6}$"
       },
-      "InitializationStrings": []
+      "InitializationStrings": [
+		"200"
+	  ]
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/Huion/H610 Pro.json:28]: Don't modify an old identifier, duplicate it and make changes.

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

File: OpenTabletDriver/Configurations/Huion/H610 Pro.json (hunk 1 above)
Comment (X9VoiD) [reply OpenTabletDriver/Configurations/Huion/H610 Pro.json:28]: The old identifier was suspicious anyway so imo this change is fine. Index 200 never had any value before, and InputReportLength 4 is weird

Comment (InfinityGhost) [reply OpenTabletDriver/Configurations/Huion/H610 Pro.json]: Hmm understandable

File: OpenTabletDriver/Configurations/Huion/H610 Pro.json [hunk 2]
```diff
@@ -52,6 +52,31 @@
...
+      "OutputReportLength": null,
+      "ReportParser": null,
+      "FeatureInitReport": null,
+      "OutputInitReport": null,
+      "DeviceStrings": {
+        "200": "HVAN",
+        "121": "HA60"
+      },
+      "InitializationStrings": []
     }
```
Comment (X9VoiD) [inline OpenTabletDriver/Configurations/Huion/H610 Pro.json]: As reviewed, this is not needed anymore and should be removed, revert the commit with `git reset HEAD~1` then `git push --force` to delete it without extra "revert" commit.

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**